.pipelines/cs-ci-build.yaml

```diff
@@ -26,6 +26,10 @@ steps:
   inputs:
     signType: $(signType)
 
+- task: NuGetToolInstaller@1
+  inputs:
+    versionSpec: '6.4.0'
```

Comment:
`'6.4.x'` is probably better, that way it can pick up patch updates which may include security fixes.

Reply:
Good point - thanks!